xarray/tests/test_plot.py

```diff
@@ -2897,7 +2897,7 @@ def test_datetime_hue(self) -> None:
         ds2["hue"] = pd.date_range("2000-1-1", periods=4)
         ds2.plot.scatter(x="A", y="B", hue="hue")
 
-        ds2["hue"] = pd.timedelta_range("-1D", periods=4, freq="D")
+        ds2["hue"] = pd.timedelta_range("-1D", periods=4, freq="D", unit="ns")  # type: ignore[call-arg,unused-ignore]
```

Comment:
This type error has been fixed upstream, but we currently pin to an earlier version of pandas-stubs due to another issue: https://github.com/pydata/xarray/issues/10110.